java/ql/src/Security/CWE/CWE-1104/MavenPomDependsOnBintray.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p><a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/">Bintray and JCenter are shutting down on May 1st, 2021</a>.
```

Comment:
They have updated that blog post to say that they are shutting down for new package versions on that date, but they will continue serving packages till February 1st, 2022. So this text should be updated.